packages/design-system/src/components/OcTextInput/OcTextInput.spec.ts

```diff
@@ -5,6 +5,32 @@ const defaultProps = {
   label: 'label'
 }
 
+class basicPasswordPolicy {
```

Comment:
Can you work with mocks here? I'd prefer that over implementing it by yourself here. E.g.:

```
const policyMock = mock<PasswordPolicy>()
policyMock.missing.mockReturnValue()
```